internal/pkg/service/stream/storage/statistics/repository/fixtures/stats_open_snapshot_001.txt

```diff
@@ -0,0 +1,9 @@
+<<<<<
+storage/stats/local/123/456/my-source/my-sink/2000-01-01T19:00:00.000Z/my-volume/2000-01-01T20:00:00.000Z/_open
+-----
+{
+  "slicesCount": 1,
+  "firstRecordAt": "",
+  "lastRecordAt": ""
+}
```

Comment:
The UTCTime/time.Time is a struct, so the `omitempty` cannot be used.

It could be solved, but I'm not going to waste time on it.